codespell_lib/data/dictionary.txt

```diff
@@ -2378,6 +2380,8 @@ chasnge->change, changes,
 chasr->chaser, chase,
 chatacter->character
 chatacters->characters
+chatch->catch
+chatched->caught
```

Comment:
Possibly add chatted as a suggestion.

Does catched exist as a typo?

Reply:
```
$  echo "catched" | codespell -
1: catched
	catched ==> caught
```

Reply:
Sorry, I was being lazy

Reply:
> Possibly add chatted as a suggestion.  

I think this is a bit of a strech. Perhaps @larsoner can weigh in?

Reply:
It's quite a few mistyped letters I'd agree, but if you're going for people misspelling things, as you are with the end result here being caught, I don't think it's a huge leap. If they're unsure of how to spell it, we ought to offer the other word they might be trying to spell, so they don't get even more confused.

Reply:
No opinion either way